notifications/notification.py

```diff
@@ -1,27 +1,31 @@
-# -*- coding:utf-8 -*-
-import logging
-import smtplib
-import threading
+# -*- coding: utf-8 -*-
+
 import time
 from datetime import datetime, timedelta
+
+import jinja2
```

Comment:
is this requirement included in flask? or do we need to add it to requirements.txt?

Reply:
Yes, most likely it needs to be added. But Python didn't show an error for me, maybe it's an indirect dependency of flask? Does Python support something like this?

Reply:
yeah, I'm pretty sure it's a dependency of flask.